src/main/resources/hudson/tasks/test/AbstractTestResultAction/show-failures.js

```diff
@@ -0,0 +1,33 @@
+/**
+* Displays all the tests failures, and hides the link allowing to display them from UI.
+*/
+function showFailures() {
+    // Displaying all the hidden elements from the page (those are the failed tests)
+    let hiddenElements = document.getElementsByClassName("hidden");
+
+    // DEV MEMO:
+    // hiddenElements is not an array but an HTMLCollection.
+    // To allow using forEach, we need an array, so I'm using the spread operator below to get that.
+    [...hiddenElements].forEach(element => { element.style.display = ""; });
+
+    // Now hiding the link from UI allowing to show all failed tests
+    let showFailuresLink = document.getElementById("showLink");
+    showFailuresLink.style.display = "none";
+}
+
+// Adding an onclick listener to the link in UI allowing to display all failed tests
+// DEV MEMO:
+// We are doing it after DOM content is loaded as a good practice to ensure we are not slowing down
+// the page rendering. In that particular situation the addition of the onclick handler shouldn't
+// really impact the page performances, but rather stick with good practices.
+
+document.addEventListener('DOMContentLoaded', (event) => {
+
+  // Retrieving the link from UI allowing to show all failed tests
+  // Note: we are retrieving the link by its ID to match how it was already done
+  // in the showFailures method above.
+  const showFailuresLink = document.getElementById("showLink");
```

Comment:
Note:

The Jira issue ([JENKINS-69656](https://issues.jenkins.io/browse/JENKINS-69656)) mentioned:

> Be careful, the id "showLink" is perhaps too generic (could be present on other elements in the same page. Adding a className like "showFailuresLink" and using it as selector could be a good idea.

But the original piece of code was actually using `document.getElementById("showLink")` to identify the link, so I considered it safe to rely on the very same identification for adding the `onclick` handler.

If you feel like it's not good enough, I'm happy to change it, but the current proposal matches the original behavior of the source code.

Reply:
Perhaps a matter of luck, but we have several occurrences of `showLink` being used as id: https://github.com/search?l=XML&q=org%3Ajenkinsci+showLink&type=Code

Reply:
Do you want me to change it in that PR with something unique (both in the `onclick` and in the `showFailures` function)? 

Or should I create another issue to fix it? (or ignore it?)